packages/components/src/sandbox/index.native.js

```diff
@@ -165,7 +165,7 @@ function Sandbox( {
 					<script
 						type="text/javascript"
 						dangerouslySetInnerHTML={ {
-							__html: observeAndResizeJS,
+							__html: customJS || observeAndResizeJS,
```

Comment:
The WP embed uses a different method to observe for size changes so we have to provide a way to inject custom Javascript.